Assets/InputSimulationHelpGuide.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    public class InputSimulationHelpGuide : MonoBehaviour
+    {
+        [SerializeField]
+        [Tooltip("Additional distance on top of sphere cast radius when pointer is considered 'near' an object and far interaction will turn off")]
+        private GameObject ShortcutTip = null;
+
+        [SerializeField]
+        [Tooltip("Additional distance on top of sphere cast radius when pointer is considered 'near' an object and far interaction will turn off")]
+        private bool displayShortcutTipOnStart = true;
+
+        [SerializeField]
+        [Tooltip("Additional distance on top of sphere cast radius when pointer is considered 'near' an object and far interaction will turn off")]
+        private GameObject HelpGuideVisual = null;
+
+        [SerializeField]
+        [Tooltip("Additional distance on top of sphere cast radius when pointer is considered 'near' an object and far interaction will turn off")]
```

Comment:
This is a draft, but just FYI all these fields are missing correct tooltips and the class is missing a comment.

Additionally, following the principle to allow things to be configured both in editor and in code, make sure that this shortcut can be instantiated and configured from code via properties.